models/DataObject/ClassDefinition/Data/Extension/RelationFilterConditionParser.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * @category    pimcore
+ * @date        25/06/2021
+ * @author      Filip Szenborn <fszenborn@divante.co>
+ */
+declare(strict_types=1);
```

Comment:
please follow coding standards defined here https://github.com/pimcore/pimcore/blob/10.x/.php-cs-fixer.dist.php

Author information will be stored with the commit
```suggestion
declare(strict_types=1);

/**
 * Pimcore
 *
 * This source file is available under two different licenses:
 * - GNU General Public License version 3 (GPLv3)
 * - Pimcore Commercial License (PCL)
 * Full copyright and license information is available in
 * LICENSE.md which is distributed with this source code.
 *
 *  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
 *  @license    http://www.pimcore.org/license     GPLv3 and PCL
 */
```